content/sensu-go/5.21/sensuctl/reference.md

```diff
@@ -83,6 +83,29 @@ Run:
 sensuctl user change-password --interactive
 {{< /highlight >}}
 
+You must specify the user's current password to use the `sensuctl user change-password` command.
+
+#### Reset a user password
+
+To reset a user password without specifying the current password, run:
+
+{{< highlight shell >}}
+sensuctl user reset-password USERNAME --interactive
+{{< /highlight >}}
+
+You must have admin permissions to use the `sensuctl user reset-password` command.
+
+#### Generate a password hash
+
+To generate a password hash for the given cleartext password, run:
+
+{{< highlight shell >}}
+sensuctl user hash-password
+{{< /highlight >}}
+
+The `sensuctl user hash-password` command adds a field to the `User` resource, `password_hash`, whose value is a hash for the user's password.
```

Comment:
The `sensuctl user hash-password` command creates a bcrypt hash of a given password.

(It does not add a field to the User resource, although one was added for this release)